external/sql/storm-sql-runtime/src/test/org/apache/storm/sql/TestUtils.java

```diff
@@ -60,6 +62,25 @@ public static String result(String accumulator) {
   }
 
 
+  public static class TopN {
+    public static PriorityQueue<Integer> init() {
+      return new PriorityQueue<>();
+    }
+    public static PriorityQueue<Integer> add(PriorityQueue<Integer> accumulator, Integer n, Integer val) {
+      if (accumulator.size() >= n) {
```

Comment:
yes, it will throw NPE if n is 0, since it tries to peek from empty queue. will address this.